File: Gigante Rama Sergio/Assets/Gigante.js
#pragma strict

var movimientoIzq: KeyCode;
var movimientoDer: KeyCode;
var velocidad: int;
 
function Start () {
		
}

function Update () {

	if(Input.GetKey(movimientoIzq)){
		GetComponent.<Rigidbody2D>().velocity.x = -velocidad;
	}
	
	if(Input.GetKey(movimientoDer)){
		GetComponent.<Rigidbody2D>().velocity.x = velocidad;
	}
}

File: Gigante Rama Sergio/Assets/Scrypts/Gigante.cs
using UnityEngine;
using System.Collections;

public class Gigante : MonoBehaviour
{
	public KeyCode movimientoIzq;
	public KeyCode movimientoDer;
	public float velocidad;
	private Rigidbody2D gigante;

	void Start () {
		gigante = GetComponent<Rigidbody2D>();
	}
	
	void Update () {

		Vector2 tempVelocity;
		tempVelocity = gigante.velocity;

		if(Input.GetKey(movimientoIzq)){
			tempVelocity.x = velocidad;
			gigante.velocity = tempVelocity;
		}
		
		if(Input.GetKey(movimientoDer)){
			tempVelocity.x = -velocidad;
			gigante.velocity = tempVelocity;
		}
	}
}